enterprise/j2eeapis/nbproject/project.properties

```diff
@@ -17,3 +17,5 @@
 
 is.autoload=true
 release.external/deployment-api-1.2-rev-1.jar=modules/ext/jsr88javax.jar
+
+sigtest.public.packages=javax.enterprise.deploy
```

Comment:
this restricts the sig test to only the listed package prefix?

Reply:
Yes. That's the idea, as the recursive behavior was not working prior 1.6. They checked only the listed part of the package hierarchy. So adding these mimics the former behavior.